src/app/domains/domains.service.spec.ts

```diff
@@ -19,33 +19,64 @@ describe('DomainsService', () => {
     TestBed.configureTestingModule({})
     service = new DomainsService(new AuthService(httpClientSpy as any, routerSpy as Router), httpClientSpy as any)
   })
+  const domainResponse = {
+    data: [{
+      profileName: 'testDomain',
+      domainSuffix: 'testDomain.com',
+      provisioningCert: 'domainCertLongText',
+      provisioningCertPassword: 'password',
+      provisioningCertStorageFormat: 'string'
+    }],
+    totalCount: 1
+  }
+
+  const domainReq = {
+    profileName: 'testDomain',
+    domainSuffix: 'testDomain.com',
+    provisioningCert: 'domainCertLongText',
+    provisioningCertPassword: 'password',
+    provisioningCertStorageFormat: 'string'
+  }
 
   it('should be created', () => {
     expect(service).toBeTruthy()
   })
 
-  it('should return array of domains when get all domains called', (done: DoneFn) => {
-    const domainResponse = {
-      data: [{
-        profileName: 'testDomain',
-        domainSuffix: 'testDomain.com',
-        provisioningCert: 'domainCertLongText',
-        provisioningCertPassword: 'password',
-        provisioningCertStorageFormat: 'string'
-      }],
-      totalCount: 1
-    }
+  it('should return array of 25 domains when get all domains called ', () => {
+    httpClientSpy.get.and.returnValue(of(domainResponse))
+
+    service.getData({ pageSize: 25, startsFrom: 0, count: 'true' }).pipe().subscribe(response => {
+      expect(response).toEqual(domainResponse)
+    })
+    const params = httpClientSpy.get.calls.allArgs()[0][0].split('?')[1]
+    expect('$top=25&$skip=0&$count=true').toEqual(params)
```

Comment:
the expectation is reversed -- the order should be expect(actual).toEqual(expected)

Reply:
done